src/main/java/core/basesyntax/model/Cat.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.model;
+
+public class Cat {
+    private int age;
+    private String name;
+    private String ownerName;
+
+    @Override
+    public String toString() {
+        return "Cat{"
+                +
+                "age=" + age
+                +
+                ", name='" + name + '\''
+                +
+                ", ownerName='" + ownerName + '\''
+                +
+                '}';
```

Comment:
```suggestion
                + "age=" + age
                +  ", name='" + name + '\''
                +  ", ownerName='" + ownerName + '\''
                + '}';
```
don't leave the plus one on the line